packs/sky-kings-tomb-bestiary/ashrin.json

```diff
@@ -823,16 +823,16 @@
                     "value": 20
                 },
                 "damageRolls": {
-                    "FSsWqq1nqzSNAwRe": {
-                        "damage": "3d8+5",
```

Comment:
I assume the Moderate Alchemist's Fire and Moderate Sulfur Bomb damage was an error - I can't find anything in the stat block that would modify it.